pkgs/servers/nosql/influxdb2/default.nix

```diff
@@ -0,0 +1,125 @@
+{ buildGoModule
+, buildGoPackage
+, fetchFromGitHub
+, git
+, go-bindata
+, lib
+, llvmPackages
+, mkYarnPackage
+, pkg-config
+, rustPlatform
+, ... }:
+let
+  version = "2.0.2";
+  libflux_version = "0.95.0";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "influxdata";
+    repo = "influxdb";
+    sha256 = "1vv0f5zpxl7s6rsadhsj25zxk04m7a0jzghrbzynjv7xv7ml90r7";
+    # The build process requires `git rev-parse HEAD` to work, so we
```

Comment:
Wouldn't it be easier to patch the usage of `git rev-parse HEAD` to `echo $rev`?

Reply:
Or another way is to provide a fake `git` binary that outputs the $rev when called.

Reply:
Do we have access to $rev? The source checkout was based on a tag, but we need to embed the corresponding commit hash. Does fetchFromGitHub do magic to make the rev available to us?

Reply:
Updated to pass in the rev explicitly (i.e. maintainer has to update the hash when updating the version). This eliminates all uses of `git` and `fetchGit`.